governance/assessments/projects/kubeedge/2024-06-12.md

```diff
@@ -0,0 +1,178 @@
+
+The project includes the following sub-projects, plugins, and other notable divisions:
+
+| Area | Ownership and Operation | Standing Bodies | Project Alignment | Notes |
+|:-----|:-----------------------:|:---------------:|:------------------|:---|
+| SIG AI | Complete | Complete | Complete | |
+| SIG Device IOT | Complete | Complete | Complete | |
+| SIG MEC | Partial | Missing | Partial | Not set up yet? |
+| SIG Network | Complete | Complete | Complete | |
+| SIG Node | Partial | Partial | Partial | No chairs, meetings |
+| SIG Release | Partial | Partial | Partial | No real meetings or docs |
+| SIG Robotics | Complete | Complete | Complete | |
+| SIG Scalability | Partial | Partial | Complete | Not fully staffed |
+| SIG Security/Security Team | Partial | Partial | Partial | Two groups are not reconciled, fully staffed |
+| SIG Testing | Partial | Partial | Partial | |
+
+The SIG charters do not identify if any repositories are owned by particular SIGs.
+
+### Operation
+
+#### Transparency and freshness
+
+The project has extensive governance documentation which attempts to define all aspects of governance for the project clearly for contributors and the public.  Much of this textual documentation is recently updated, within the last 6-8 months and there is evidence of a sincere effort to keep it up to date.
+
+A lot of project decision-making seems to happen at the frequent and well-attended community and SIG meetings.  However, these meetings do not have good notetaking, and the video recordings are not up to date.  This means that project members who cannot directly attend the meetings may be left out of decision making.
+
+How PRs (and roadmap items) get approved is also not well-defined. KubeEdge uses Prow but does not have docs for the approval process or explain Prow to contributors. Clearly the project has an efficient process, because PRs are handled and closed or merged quickly, so this seems like it's just a matter of writing down existing practices.
+
+#### Governance Drift
+
+Governance Drift can occur when the executed and observable governance of a project deviates from the documented governance of the project.
+
+Because of the overhaul of project governance and the institution of the TSC, there is currently governance drift in terms of actual leadership structures not yet matching statutory ones.  See Evolution below.
+
+#### Ownership
+
+The project's ownership evaluation did not leverage Sheriff, the CNCF GitHub permission auditing tool.
+
+KubeEdge uses Prow with OWNERS files in all repos.  These OWNERs files seem to mostly match the expressed leadership of the project, although the current governance transition makes that hard to tell. It's also not clear where a specific SIG should be the owners of a repository.
+
+KubeEdge does not use Sheriff or Peribolos in order to enforce community GitHub-level permissions.  Since they're already on Prow, Peribolos might be an easier implementation.
```

Comment:
```suggestion
KubeEdge does not use Sheriff or Peribolos in order to enforce community GitHub-level permissions. Since they're already on Prow, Peribolos might be an easier implementation.
```